templates/eap-cd-amq-persistent-s2i.json

```diff
@@ -606,7 +606,8 @@
                                             "-c",
                                             "/opt/eap/bin/readinessProbe.sh"
                                         ]
-                                    }
+                                    },
+                                    "initialDelaySeconds": 10
```

Comment:
Note: for the record. The old behavior baked into probe shell script would wait 5 secs before probing and then the probe would loop retrying with a 5 sec pause between attempts. In practice we don't observe EAP containers being ready in less than 10 secs anyway, so using 10 hear avoid spurious initial failures without adding a pointless delay.